### DISCUSSION TITLE
Support for using self generated certificates

### DISCUSSION THREAD

**Discussion by nitchimon** [projectdiscovery/interactsh#169] [Q&A] [answered]
When you use -domain, it tries to register a certificate through letsencrypt.com Can I use an internally generated Certificate where we can generate our own Cert?

**Comment by ehsandeep** [comment]
We are tracking this at https://github.com/projectdiscovery/interactsh/issues/157

**Reply by godoppl** [reply]
I made a little script/tutorial for this here: #219

**Comment by ehsandeep** [comment, marked as answer]
@nitchimon this is now supported with latest release - https://github.com/projectdiscovery/interactsh#custom-ssl-certificate
